src/main/java/net/minecraftforge/event/brewing/RegisterBrewingFuelEvent.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.event.brewing;
+
+import net.minecraft.world.item.Item;
+import net.minecraftforge.common.BrewingFuelValues;
+
+public class RegisterBrewingFuelEvent extends net.minecraftforge.eventbus.api.Event 
+{
+    
```

Comment:
Our new code style is to have braces on the same line, so please do that.

Reply:
Understood. I was wondering if the "how to contribute to forge" document I found might be a little out of date... guess so. It's [here](https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge) if you want to take a look.